app/models/supporting_evidence_attachment.rb

```diff
@@ -3,6 +3,8 @@
 # Files uploaded as part of a form526 submission that will be sent to EVSS upon form submission.
 # inherits from ApplicationRecord
 class SupportingEvidenceAttachment < FormAttachment
+  # [wipn8923] this is problematic... SupportingEvidenceAttachmentUploader inherits from EVSS logic, so this will
+  # need to respect the flipper, however, we don't have access to the flipper at this level
```

Comment:
Fixed this inheritance issue in my branch

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/13835/commits/7e5c4658bb3877539f4f9fcad82cae95f5421bcd